### PR TITLE
added python -X options support.

### PR DESCRIPTION
**Pull request by xen**
One of the recommended way to turn on debug mode for development is using python `-X dev` parameter. This PR adds support for this parameter for spawn processes. Also, I updated syntax and dropped old python version support.

### CODE REVIEW

**Comment by xen** [comment]
Ping?

File: aioreloader/_contents.py [hunk 1]
```diff
@@ -119,16 +106,22 @@ def reload():
     if reload_hook is not None:
         reload_hook()
 
-    if sys.platform == 'win32':
-        subprocess.Popen([sys.executable] + sys.argv)
+    xopt = []
+    if sys._xoptions:
+        for k, v in sys._xoptions.items():
+            if type(v) == bool:
+                xopt.append(f"-X{k}")
```
Comment (and800) [inline aioreloader/_contents.py:113]:
```suggestion
                xopt.append("-X{}".format(k))
```

Comment (and800) [reply aioreloader/_contents.py:113]: 3.5

File: aioreloader/_contents.py [hunk 2]
```diff
@@ -119,16 +106,22 @@ def reload():
...
+            else:
+                xopt.append(f"-X{k}={v}")
```
Comment (and800) [inline aioreloader/_contents.py:115]:
```suggestion
                xopt.append("-X{}={}".format(k, v))
```

**Comment by xen** [comment]
Can you please release new version?

**Comment by and800** [comment]
`pip install aioreloader==0.3.0`

Thanks. Sorry it took me so long.

**Comment by xen** [comment]
No problem. Thank you for your product.

**Comment by and800** [comment]
actually I'm kind of surprised that people still stumble into this tool for so long. Now there are more popular tools like https://github.com/aio-libs/aiohttp-devtools

**Pull request merged**